### PR TITLE
Fe/profile ui

### PR DESCRIPTION
**Pull request by lilianaweimer**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Profile only shows email

## What is the new behavior?

- Profile includes address and description (image with `src=''` as placeholder)
- Profile includes buttons for editing profile info and image
- On click, the edit profile button toggles the `ProfileForm` component
- The `ProfileForm` component can dynamically update its own state on change

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

No interaction with backend yet, and clicking the submit button does nothing (yet!).

### CODE REVIEW

File: code/web/src/modules/user/Profile.js [hunk 1]
```diff
@@ -11,13 +11,34 @@ import { H3, H4 } from '../../ui/typography'
...
+      modalDisplay: false,
+    }
+  }
+
+  toggleModal = () => {
+    this.setState((prevState) => {
+      return { modalDisplay: !prevState.modalDisplay }
+    })
+  }
+
```
Comment (jdvolk) [inline code/web/src/modules/user/Profile.js:34]: This looks goods!

File: code/web/src/modules/user/Profile.js [hunk 2]
```diff
@@ -30,21 +51,41 @@ const Profile = (props) => (
...
-    </Grid>
+    </Grid> : null}
+    {this.state.modalDisplay ? 
+    <Grid>
+      <ProfileForm 
+        submit={this.submit}
+        toggleModal={this.toggleModal}
+      />
+    </Grid> 
+    : null}
```
Comment (jdvolk) [inline code/web/src/modules/user/Profile.js:83]: very clean

File: code/web/src/modules/user/ProfileForm.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  render() {
```
Comment (jdvolk) [inline code/web/src/modules/user/ProfileForm.js:36]: Form looks nice!!

File: code/web/src/modules/user/Profile.js [hunk 4]
```diff
@@ -30,21 +51,41 @@ const Profile = (props) => (
...
+        <p style={{ color: grey2, marginBottom: '2em' }}>{this.props.user.details.city}city, {this.props.user.details.state}state {this.props.user.details.zip}zip</p>
+        <p style={{ color: grey2, marginBottom: '2em' }}>{this.props.user.details.description}description</p>
+        <Button theme="primary" style={{ marginRight: '1em' }} onClick={this.toggleModal}>Update Profile</Button>
 
         <Link to={userRoutes.subscriptions.path}>
           <Button theme="primary">Subscriptions</Button>
         </Link>
 
-        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+        <Button theme="secondary" onClick={this.props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
```
Comment (cgaddis36) [inline code/web/src/modules/user/Profile.js:73]: nice onClick function

File: code/web/src/modules/user/ProfileForm.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    return (
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
```
Comment (cgaddis36) [inline code/web/src/modules/user/ProfileForm.js:38]: Nice work here, I've been wanting to build one of these for months

**Pull request merged**